cana/boolean_node.py

```diff
@@ -636,6 +636,43 @@ def bias(self):
 		"""
 		return sum(self.outputs) / 2**self.k
 
-
-
-
+	def c_sensitivity(self, c, mode="default", max_k=0):
+		"""give the c-sensitivity of this node's Boolean function
```

Comment:
No problem. 
So as I read on Stackoverflow, I just need to push a new commit to my branch, the push request would update automatically?